packages/flutter_tools/lib/src/android/gradle_utils.dart

```diff
@@ -38,10 +38,10 @@ const String templateKotlinGradlePluginVersion = '1.8.22';
 // so new versions are picked up after a Flutter upgrade.
 //
 // Please see the README before changing any of these values.
-const String compileSdkVersion = '34';
+const String compileSdkVersion = '35';
 const String minSdkVersion = '21';
-const String targetSdkVersion = '34';
-const String ndkVersion = '23.1.7779620';
+const String targetSdkVersion = '35';
+const String ndkVersion = '26.1.10909125';
```

Comment:
Is this ndk version updating to align with the engine?

Reply:
Yes